projects/Mallard/src/authentication/lib/Authorizer.ts

```diff
@@ -154,7 +177,8 @@ class Authorizer<T, A extends any[], C extends readonly AsyncCache<any>[]> {
                 ? ValidAttempt(this.name, attempt.connectivity, attempt.time)
                 : InvalidAttempt(
                       attempt.connectivity,
-                      'Insufficient privileges',
+                      (!isValid(attempt) && attempt.reason) ||
+                          'Insufficient privileges',
```

Comment:
Should this be capitalised?

Reply:
It's capitalised as it's literally thrown to the UI as is, rather than using some uppercase-first-letter helper. They way I modelled them is that these messages should be user-facing but, potentially that doesn't necessarily mean capitalised 🤷‍♂